### DISCUSSION TITLE
CamAPS + Companion App - Why is it not working?

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2558] [Q&A]
![320305133_1832872570401691_3039359690649465866_n](https://user-images.githubusercontent.com/51497406/207938656-25736e63-a55c-4a31-83bb-f15dfa3f6295.jpg)

![320292285_473924321335366_8463760195815093962_n](https://user-images.githubusercontent.com/51497406/207938408-1ed2f6cd-1f05-4ccf-a410-48dbce0a48a8.jpg)

**Comment by jamorham** [comment]
Is xDrip also set for mmol units?

**Reply by Sandels1986** [reply]
Yes, both are mmol units.

**Reply by jamorham** [reply]
Is this the play store or amazon store version?

**Reply by Sandels1986** [reply]
Actually I don't know. In Finland we have to download this app from the website, one member on the finnis diabetes group share the link for me.

**Reply by jamorham** [reply]
Ok well another thing to check is notification access. Normally this is in android settings -> apps and notifications -> special app access -> Notification access -> xDrip+ set to allow

**Reply by Sandels1986** [reply]
All notifications are allowed. But I just  heard today that we would have new app early next year, so I think we just use only camAPS now. It works ok in our daughters phone.

**Reply by Navid200** [reply]
What do you mean it works OK on your daughter's phone?
I thought everything you were showing so far was on your daughter's phone.  Wasn't it?

**Reply by Sandels1986** [reply]
Yes, camAPS works on my daughters phobe. Xdrip doesn't. We need xdrip so we parents can have notifications on our phones. CamAPS give us only textmessage notifications to us. And we can't follow our daughters bloodsugar when she is out of home. Xdrip give that possibily also.

**Reply by steve8x8** [reply]
@Sandels1986 can you check the "internal ID" of the APK? I wouldn't expect the Finnish version to come with yet another ID string, but you never know... Go to settings -> apps, then select the CamAPS app and find the string (depends on the Android UI: LineageOS puts it at the end of the "advanced" part while MIUI hides it behind a question mark...)

**Comment by jamorham** [comment]
I released a new nightly to address an issue which might be related. Please let me know if the Jan 17th Nightly works any better here. Thanks.
